### DISCUSSION TITLE
日本語環境のみでアップデート、インストールが出来ないトラブル対策について　How to troubleshoot issues with updates and installations that are only possible in a Japanese environment

### DISCUSSION THREAD

**Discussion by gatarou041** [lllyasviel/Fooocus#3306] [General]
現在日本語環境のみでアップデート、新規インストールが出来ない問題が発生していますがこちらのポスト『https://x.com/AonekoSS/status/1813592759045112314』を参考の上　Windowsの検索から設定の環境変数「PYTHONUTF8」を「1」で設定するとインストールやアップデートが可能になるようです

Currently, there is an issue where updates and new installations cannot be performed in Japanese environments, but please refer to this post: https:

**Comment by mashb1t** [comment]
=> https://x.com/AonekoSS/status/1813592759045112314

Thanks for sharing, hopefully helps some users who are also affected by the issue.
